pubsignals/query_test.go

```diff
@@ -310,13 +317,43 @@ func TestCheckRequest_Success(t *testing.T) {
 }
 
 func TestCheckRequest_SelectiveDisclosure_Error(t *testing.T) {
+	now := time.Now().Unix()
+	durationMin, _ := time.ParseDuration("-1m")
+	dayAndMinuteAgo := time.Now().AddDate(0, 0, -1).Add(durationMin).Unix()
```

Comment:
Could you explain, why are you using  `durationMin, _ := time.ParseDuration("-1m")`